integration_test/cases/browser/set_value_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Wallaby.Integration.Browser.SetValueTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "forms.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "set_value/3" do
+    test "accepts a query", %{page: page} do
+      page
+      |> send_keys(Query.text_field("Name"), ["Chris", :tab, "c@keathley.io"])
```

Comment:
Is there something specific here that we'd wanna use `send_keys` over `fill_in` for filling in the form fields?